heron/schedulers/src/java/com/twitter/heron/scheduler/kubernetes/HttpJsonClient.java

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.twitter.heron.common.network;
+package com.twitter.heron.scheduler.kubernetes;
```

Comment:
The `network` directory contains very fundamental network util classes for internal usage. Given this `HttpJsonClient` will be used in `schedulers` in different places only, I would suggest we put it into `schedulers-utils`.

Thus we can keep the `common` simple and clean. Let me know your idea, @kramasamy @jrcrawfo